src/test/java/com/znsio/teswiz/businessLayer/vodqa/VodqaBL.java

```diff
@@ -0,0 +1,46 @@
+package com.znsio.teswiz.businessLayer.vodqa;
+
+import com.context.TestExecutionContext;
+import com.znsio.teswiz.businessLayer.theapp.AppBL;
+import com.znsio.teswiz.entities.Platform;
+import com.znsio.teswiz.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.teswiz.runner.Runner;
+import com.znsio.teswiz.screen.vodqa.VodqaScreen;
+import org.apache.log4j.Logger;
+import org.assertj.core.api.SoftAssertions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class VodqaBL {
+    private static final Logger LOGGER = Logger.getLogger(AppBL.class.getName());
+    private final TestExecutionContext context;
+    private final SoftAssertions softly;
+    private final String currentUserPersona;
+    private final Platform currentPlatform;
+    public VodqaBL() {
+        long threadId = Thread.currentThread().getId();
+        this.context = Runner.getTestExecutionContext(threadId);
+        softly = Runner.getSoftAssertion(threadId);
+        this.currentUserPersona = SAMPLE_TEST_CONTEXT.ME;
+        this.currentPlatform = Runner.getPlatform();
+    }
+    public VodqaBL login() {
+        LOGGER.info("login(): login into the application");
+        VodqaScreen vodqaScreen = VodqaScreen.get();
+        assertThat(vodqaScreen.verifyLoginPage()).as("Login page not loaded").isTrue();
```

Comment:
this assertion is not required and can be removed, refer to ScreenAndroid explained there.